integreat_cms/cms/templates/pois/poi_form_sidebar/related_contacts_box.html

```diff
@@ -0,0 +1,28 @@
+{% extends "../../_collapsible_box.html" %}
+{% load i18n %}
+{% load widget_tweaks %}
+{% block collapsible_box_icon %}
+    message-square
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% trans "Related contacts" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    {% with poi_form.instance.contacts.all as contacts %}
+        <div>
+            <div class="help-text">
+                {% if contacts %}
+                    {% trans "This location is currently referred to in those contacts." %}
```

Comment:
```suggestion
                    {% trans "This location is currently referred to by the following contacts." %}
```
I think this would be more consistent with the German translation in the translation file